meta-oe/recipes-support/source-highlight/source-highlight_git.bb

```diff
@@ -0,0 +1,36 @@
+SUMMARY = "Syntax highlight utility"
+DESCRIPTION = "Source-highlight converts source code to formatted text with syntax highlighting."
+HOMEPAGE = "https://www.gnu.org/software/src-highlite/"
+LICENSE = "GPL-3.0-only"
+SECTION = "libs"
+LIC_FILES_CHKSUM = "file://COPYING;;md5=ff95bfe019feaf92f524b73dd79e76eb"
+
+SRCREV = "894cacd0799ca60afa359a63782729dec76cbb79"
+PV = "3.1.9+git"
+SRC_URI = "git://git.savannah.gnu.org/git/src-highlite.git;protocol=https;branch=master"
+S = "${WORKDIR}/git"
+
+inherit autotools pkgconfig
+
+DEPENDS:append = " bison-native boost"
+
+DEPENDS:append:class-target = " ${PN}-native"
```

Comment:
This doesn't work well with multilib, causes:

ERROR: Nothing PROVIDES 'lib32-source-highlight-native' (but virtual:multilib:lib32:meta-oe/meta-oe/recipes-support/source-highlight/source-highlight_git.bb DEPENDS on or otherwise requires it). Close matches:
  lib32-source-highlight
  source-highlight
  source-highlight-native

Reply:
Thanks for your feedback. I've fixed it in https://github.com/openembedded/meta-openembedded/pull/958